docs/dynamic-plugins.md

```diff
@@ -761,7 +761,7 @@ Each plugin can expose multiple API Factories and each factory is required to de
 - `importName` is an optional import name that reference a `AnyApiFactory<{}>` implementation. Defaults to `default` export.
 - `module` is an optional argument which allows you to specify which set of assets you want to access within the plugin. If not provided, the default module named `PluginRoot` is used. This is the same as the key in `scalprum.exposedModules` key in plugin's `package.json`.
 
-There are a set of [API factories](../packages/app/src/apis.ts) already initialized by the Developer Hub application shell. These API factories can be overridden by an API factory provided by a dynamic plugin by specifying the same API ref ID, for example a dynamic plugin could export the following `AnyApiFactory<{}>` to cater for some specific use case:
+There are a set of [API factories](https://github.com/janus-idp/backstage-showcase/blob/main/packages/app/src/apis.ts) already initialized by the Developer Hub application shell. These API factories can be overridden by an API factory provided by a dynamic plugin by specifying the same API ref ID, for example a dynamic plugin could export the following `AnyApiFactory<{}>` to cater for some specific use case:
```

Comment:
:thinking: the problem with absolute references is that if you're reading the 1.3 docs they'll link you to the 1.4 docs... is that ideal?

Reply:
Hmmm yeah that's a really good point 🤔 We should be prioritizing having the links work in TechDocs over Github right? If so I can update it to the correct relative reference in with TechDocs but it would no longer work with Github markdown viewer.

Reply:
Sorry I thought this was about linking to an image or another markdown file. I don't think there is a way to open a typescript file with techdocs without providing the actual GitHub url. We do do this in a few other locations already ([here](https://github.com/janus-idp/backstage-showcase/blob/5e4984500b9124784481caf4ed7ce5a99e9fdf84/docs/dynamic-plugins.md?plain=1#L133)) as well.

Reply:
Retracting my comment as I may be solving a problem that isn't. +1, LGTM.